llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -7166,8 +7166,11 @@ bool SPIRVToLLVM::translate(ExecutionModel entryExecModel, const char *entryName
   // ; buffer${i}Stream is the vertex StreamId that writes to buffer i, or -1 if that particular buffer is unused (e.g.
   // StreamId in EmitVertex)
   if (enableXfb) {
-    const unsigned bufferCount = 8;
-    std::array<int, 8> xfbState{-1, 0, -1, 0, -1, 0, -1, 0};
+    static const unsigned MaxXfbBuffers = 4;
+    std::array<unsigned, 2 * MaxXfbBuffers> xfbState{SPIRVID_INVALID, 0,  // xfbBuffer[0] -> <streamId, xfbStride>
+                                                     SPIRVID_INVALID, 0,  // xfbBuffer[1] -> <streamId, xfbStride>
+                                                     SPIRVID_INVALID, 0,  // xfbBuffer[2] -> <streamId, xfbStride>
+                                                     SPIRVID_INVALID, 0}; // xfbBuffer[3] -> <streamId, xfbStride>
```

Comment:
Should this be InvalidValue for consistency? It's not a SPIRV-ID...

Reply:
Yes, you are right. In SPIR-V translator, we have legacy problems in using SPIRVID_INVALID. It is better to always use InvalidValue to make it clear.